tests/manager/manager-executeCommand_error-003.phpt

```diff
@@ -17,5 +17,5 @@ echo throws(function() use($manager, $command) {
 <?php exit(0); ?>
 --EXPECTF--
 OK: Got MongoDB\Driver\Exception\ConnectionTimeoutException
-No suitable servers found (`serverSelectionTryOnce` set): [connection refused calling ismaster on '%s']
+No suitable servers found (`serverSelectionTryOnce` set): %s
```

Comment:
AFAIK, ["connection refused"](https://github.com/mongodb/mongo-c-driver/blob/1.11.0/src/libmongoc/src/mongoc/mongoc-async.c#L145) comes from libmongoc encountering a `POLLHUP` error on the socket during monitoring.

I found no record of libmongoc assigning a "connection failed" string via `bson_set_error()` in its topology scanner or async components, although there are some similar strings (e.g. ["connection closed"](https://github.com/mongodb/mongo-c-driver/blob/1.11.0/src/libmongoc/src/mongoc/mongoc-async.c#L148), ["connection timeout"](https://github.com/mongodb/mongo-c-driver/blob/1.11.0/src/libmongoc/src/mongoc/mongoc-async.c#L178)). It's possible that "connection failed" comes from some other source outside of libmongoc (perhaps a function that converts a socket error to a string).

In any event, I don't see the harm in relaxing this error message, as we're mainly testing for a ConnectionTimeoutException and server selection failure.